dynatrace-aws-logs.sh

```diff
@@ -132,7 +139,7 @@ function generate_test_log()
 "cloud.provider": "aws",
 "content": "AWS Log Forwarder installation log",
 "severity": "INFO",
-"log.forwarder.setup": "$STACK_NAME"
+"cloud.log_forwarder": "$(get_cloud_log_forwarder)"
```

Comment:
This test log is sent from your terminal and there is actually no log forwarder, so I would skip this metadata attribute here altogether

Reply:
The initial message without `"cloud.log_forwarder" `at all, yes?